packages/typespec-python/src/emitter.ts

```diff
@@ -194,26 +195,28 @@ function getEffectiveSchemaType(context: SdkContext, type: Model): Model {
     return type;
 }
 
-function isEmptyModel(type: EmitterType): boolean {
-    // object, {}, Model{} all will be treated as empty model
+function isEmptyModel(program: Program, type: EmitterType): boolean {
```

Comment:
nit: we should make `isEmptyModel` take in an `SdkContext`, it's more consistent